### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by SAZZAD1Q2**
Create a app.rb file that will serve as a console app entry point
In main.rb I define the entry point, this will be a method called main that is invoked at the end of the file

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,9 +1,59 @@
...
+end
+
+def main
+  app = App.new
+
+  loop do
+    display_options
+    choice = gets.chomp.to_i
+    handle_choice(app, choice)
+    puts "\n"
```
Comment (abdulhamiid) [inline main.rb]: When I started your app, I could not access all the available options of the app 👇 

image

According to the [project requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md), our console app should behave in the same way as it is in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) 😊🛠.

image

Example video
image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    has_permission = gets.chomp
+    permission_values = %w[n N]
+    student = Student.new(age, name, permission_values.include?(has_permission))
+    @people.push(student)
+  end
```
Comment (abdulhamiid) [inline app.rb]: I encountered `ArgumentError` while creating a person.

tempsnip

Please fix this issue so that the console app works as expected.

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @SAZZAD1Q2 👋, 
Alt Text
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Build interactive console apps.
- ✔️ Run a program using the command line.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by shella12 — APPROVED** [review]
Hi @SAZZAD1Q2, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Console app works as expected ✔️ 
- Fixed error form `Create Person` option 👍 
- Implemented all requirements 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**